app/MMB/src/scenes/tripOverview/TimelineTitle.js

```diff
@@ -0,0 +1,80 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { graphql, createFragmentContainer } from '@kiwicom/mobile-relay';
+import { Translation, DateFormatter } from '@kiwicom/mobile-localization';
+import { StyleSheet, Color, Text } from '@kiwicom/mobile-shared';
+import idx from 'idx';
+
+import type { TimelineTitle as TimelineTitleDataType } from './__generated__/TimelineTitle.graphql';
+
+type Props = {|
+  +data: TimelineTitleDataType,
+|};
+
+function TimelineTitle(props: Props) {
+  const data = props.data;
+  const localTime = idx(data, _ => _.localTime);
+  const cityName = idx(data, _ => _.airport.city.name);
+  const iataCode = idx(data, _ => _.airport.locationId);
+
+  return (
+    <View style={styleSheet.row}>
+      <View style={styleSheet.dateTime}>
+        {localTime && (
```

Comment:
This is not necessarily `android-safe` should localTime ever be 0, or anything that would be interpreted as text && false, it could crash android.

Reply:
Can you please elaborate more on the Android safeness here? Isn't output of `idx` call always `null`, `undefined` or the expected value (string date in this case)?

How to write it better?

Reply:
If localTime would return '' (not sure if it is a possible value from graphql) it results in crash on android. 

having a fallback value idx(...) || '', then {localTime !== '' && ...
or {Boolean(localTime) && ...

would prevent this

Reply:
> not sure if it is a possible value from graphql

I hope not. The output type is `DateTime` and it should fail otherwise. Anyway - fixed... :)